demisto_sdk/commands/lint/linter.py

```diff
@@ -1306,3 +1346,285 @@ def _get_commands_list(self, script_obj: dict):
         except Exception:
             logger.debug("Failed getting the commands from the yml file")
         return commands_list
+
+    def _is_native_image_support_script(
+        self,
+        native_image: str,
+        supported_native_images: Set[str],
+        script_id: str,
+    ) -> bool:
+        """
+        Gets a native image name (flag) and checks if it supports the integration/script that lint runs on.
+
+        Args:
+            native_image (str): Name of the Native image to run on.
+            supported_native_images (set): A set including the names of the native images that support the
+                                           script/integration that lint runs on.
+            script_id (str): The ID of the integration/script that lint runs on.
+        Returns (bool): True - if the native image supports the integration/script that lint runs on.
+                        False - Otherwise.
+        """
+        if native_image not in supported_native_images:
+            # Integration/Script isn't supported by the requested native image
+            logger.info(
+                f"Skipping checks on docker for {native_image} - {script_id} is not supported by the "
+                f"requested native image: {native_image}."
+            )
+            return False
+
+        return True
+
+    def _get_native_image_name_from_config_file(
+        self,
+        docker_image_flag: str,
+    ) -> Union[str, None]:
+        """
+        Gets a native docker image flag and returns its mapped native image name from the
+        'docker_native_image_config.json' file.
+
+        If the requested docker image flag doesn't have a mapped native image in the config file - write to the logs.
+
+        Args:
+            docker_image_flag (str): Requested docker image flag.
+        Returns (None): None
+        """
+        native_image_config = (
+            NativeImageConfig()
+        )  # parsed docker_native_image_config.json file (a singleton obj)
+
+        native_image = native_image_config.flags_versions_mapping.get(docker_image_flag)
+        if native_image:
+            return native_image
+
+        else:
+            # The requested native image doesn't exist in the native config file
+            logger.info(
+                f"Skipping checks on docker for '{docker_image_flag}' - The requested native image: "
+                f"'{docker_image_flag}' is not supported. For supported native image versions please see:"
+                f" 'Tests/docker_native_image_config.json'."
```

Comment:
i guess we should make it constant cause we use it in multiple places (used also in the native image module)

Reply:
Used the existing const in this [commit](https://github.com/demisto/demisto-sdk/pull/2613/commits/13caa99d420f31318bfd48860b57d39a0d70d8ce).